packages/ariakit/src/checkbox/checkbox.tsx

```diff
@@ -60,10 +60,16 @@ function isNativeCheckbox(tagName?: string, type?: string) {
  * ```
  */
 export const useCheckbox = createHook<CheckboxOptions>(
-  ({ state, value, checked: checkedProp, defaultChecked, ...props }) => {
+  ({
+    state,
+    value: attributeValue,
```

Comment:
When debugging this issue, we ran into confusion with the different "values" within this hook. We've updated the naming here to hopefully be more clear that this is the "value" attribute on the element itself.

This was done in a separate commit from the fix, so if you would like to revert this change we can.